.travis.yml

```diff
@@ -35,13 +38,15 @@ install:
   # from https://stackoverflow.com/questions/43918874/how-to-increase-no-activity-wait-time-in-travis-ci
   - "travis_wait 30 sleep 1800 &"
   - docker build --build-arg "version=$NPM_TAG" --build-arg "NODE_VERSION=$NODE_VERSION" --build-arg "GITHUB_TOKEN=$GITHUB_TOKEN" . -t "$IMAGE_TAG" --no-cache
+#  - docker build --build-arg "version=$NPM_TAG" --build-arg "NODE_VERSION=$NODE_VERSION" --build-arg "GITHUB_TOKEN=$GITHUB_TOKEN" --build-arg "NODE_OPTIONS=$NODE_OPTIONS" . -t "$IMAGE_TAG" --no-cache
```

Comment:
I'm not sure commenting out this command is appropriate, it means that other images cannot use the 
command any longer.

Reply:
The change is actually an addition by me to be able to pass NODE_OPTIONS via travis settings to the container. This was originally not possible. This is not needed anymore and I will rollback this modification.

Reply:
The change was reverted. During testing by Travis I am getting failed tests:

`[05:25:34]  	Saved screenshot: ERROR_chrome_2019-07-16T05-25-34.888Z.png`

How do I access these images?

Reply:
> How do I access these images?

Not sure if it's possible, when running on Travis. However the `wdio` test suite we run ATM is very superficial, and mostly tests that the Theia application is able to start, serve a front-end and has the basic expected workbench layout/views. 

You can try to run it locally and then you'd be able to access the screenshot. Quickly: 
- build an image, start it using external port `4000`: `docker run --init -d -p 0.0.0.0:4000:3000 <$IMAGE_TAG>`
- `cd <repo root>/tests`
- `yarn; yarn test-app-$IMAGE_NAME`  # where $IMAGE_NAME = e.g. `theia-java`, `theia-full`, ...

See file `.travis.yml` in the repo's root for more details

Reply:
Ok. See later comment on addition of a "commit_screenshots.sh" script.